src/cargo-kani/src/call_goto_instrument.rs

```diff
@@ -79,7 +79,7 @@ impl KaniSession {
     fn undefined_functions(&self, file: &Path) -> Result<()> {
         let args: Vec<OsString> = vec![
             "--generate-function-body-options".into(),
-            "assert-false".into(),
+            "assert-false-assume-false".into(),
```

Comment:
Please add a test that checks this behavior, e.g.:
```rust

extern "C" {
    fn missing_function() -> u32;
}

#[kani::proof]
fn main() {
    unsafe {
        let x = missing_function();
        assert!(x == 5);
    }
}
```
The assert should be reported as unreachable. Without your change, it fails.

Reply:
Sure.